mbq/metrics/__init__.py

```diff
@@ -33,7 +36,7 @@
 )
 
 
-def init(service: str, env: mbq.env.Environment, constant_tags=None):
+def init(service: str, env: mbq.env.Environment, constant_tags: Tags = []):
```

Comment:
For everywhere you switch it from `=None` to `=[]`: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
well shoot